fm_eventmanager/settings.py.docker

```diff
@@ -238,3 +240,7 @@ MQTT_LOGIN = {
     "username": "printing",
     "password": "secret",
 }
+
+ENVIRONMENT_NAME = os.getenv('ENVIRONMENT_NAME', "Development server")
```

Comment:
Maybe "Production" should be the default?

Reply:
updated default value